tests/pip/install.fmf

```diff
@@ -5,11 +5,9 @@ require:
     - python3
     - python3-devel
 tier: null
-adjust+:
-  - when: distro >= fedora-43
-    enabled: false
-    because: |
-      Python 3.14 wheels for cffi are not available yet
+
+environment:
+    PIP_PRE: 1
```

Comment:
Hmm, this thing seems to work and `cffi` seems to be the only blocker there, or at least the other ones also have a pre-release. We could either wait for `cffi` to have its release or make this a more permanent option for `rawhide` and `fedora-development` (though we would have to manually track the later alias).

I've tried to see if we can add a python3.14 case to the dependency, but we don't have the beta version in downstream and I don't think we can update it for both rawhide and F43 to eliminate this issue here.